README.md

```diff
@@ -61,6 +61,27 @@ Start MCP server with `sse` transport.
 CRATEDB_MCP_TRANSPORT=sse uv run cratedb-mcp
 ```
 
+# Simple Claude configuration
+To use the MCP version within Claude Desktop, you can use the following configuration:
+
+```json
+{
+  "mcpServers": {
+    "my_cratedb": {
+      "command": "uvx",
+      "args": ["--python", "3.10", "--from", "git+https://github.com/crate/cratedb-mcp", "cratedb-mcp"],
+      "env": {
+        "CRATEDB_MCP_HTTP_URL": "http://localhost:4200/",
+        "CRATEDB_MCP_TRANSPORT": "stdio"
+      }
+    }
+  }
+}
+```
+
+You might have to change `CRATEDB_MCP_HTTP_URL` to match your CrateDB instance.
+
```

Comment:
Possibly provide an example for CrateDB Cloud right away.

> You might have to change `CRATEDB_MCP_HTTP_URL` to match your CrateDB instance.
> When connecting to CrateDB Cloud, use a value like `https://admin:dZ...6LqB@testdrive.eks1.eu-west-1.aws.cratedb.net:4200/`.

Reply:
Good idea